examples/client.py

```diff
@@ -57,10 +57,11 @@
     # next time the client starts instead.
     sleep(1)
 
-# Synchronous example, insert a shutdown event
-shutdown_data = {"label": "shutdown"}
-shutdown_event = Event(timestamp=now, data=shutdown_data)
-inserted_event = client.insert_event(bucket_id, shutdown_event)
+# Synchronous example, insert an event
+event_data = {"label": "non-heartbeat event"}
+now = datetime.now(timezone.utc)
+event = Event(timestamp=now, data=shutdown_data)
```

Comment:
`data=shutdown_data` -> `data=event_data`

Reply:
Hmm weird, was pretty certain that i had tested that code but apparently not. Fixed.